### ISSUE TITLE
cmmdit

### ISSUE THREAD

**Issue by fusiming3** [Xuan-World/UniCombine#2]
代码中query我看是query = torch.cat([encoder_hidden_states_query_proj, query], dim=2)， 但是论文中您说和omnicontrol一样，但是omnicontrol是query = torch.cat([encoder_hidden_states_query_proj, query, query_condition], dim=2)。 这是我理解问题吗

**Comment by Xuan-World** [comment]
这里你的理解有误，你可以再仔细看一下代码，或者询问一些AI来帮你解读代码。
